.github/workflows/ci-pipeline.yml

```diff
@@ -46,3 +46,52 @@ jobs:
 
       - name: Deploy to Azure Web App
         run: curl -dH -X POST ${{ secrets.AZURE_WEB_HOOK }}
+
+  azure-deploy:
+    name: deploy to Azure Web App
+    runs-on: ubuntu-latest
+    needs: docker-publish
+    steps:
+      - name: Deploy to Azure Web App
+        run: curl -dH -X POST ${{ secrets.AZURE_WEB_HOOK }}
```

Comment:
Now that we're using Terraform we can stop using the old system, so we could get rid of both of these parts that trigger the AZURE_WEB_HOOK